src/webui/components/Icon/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @prettier
+ * @flow
+ */
+
+import React from 'react';
+import type { Node } from 'react';
+
+import capitalize from '../../utils/capitalize';
+import { SVG, Img, ImgWrapper } from './styles';
+import { IProps, IIconsMap } from './types';
+
+import brazil from './img/brazil.svg';
+import china from './img/china.svg';
+import india from './img/india.svg';
+import nicaragua from './img/nicaragua.svg';
+import pakistan from './img/pakistan.svg';
+import austria from './img/austria.svg';
+
```

Comment:
Typo.. a new line ❌

Reply:
the same here..

Reply:
The issue we saw yesterday on Webpack, for some reason this works on development, but those files are not hashed and not cached properly by webpack in production build.

![screen shot 2018-11-23 at 8 17 49 am](https://user-images.githubusercontent.com/558752/48931755-49aa5580-eef8-11e8-8b53-66895c2a3b11.png)

I think this configuration for `svg` won't work. We might try with other thigs. https://survivejs.com/webpack/loading/images/#loading-svgs
```
 {
        test: /\.(jpe?g|png|gif|svg)$/,
        use: [
          {
            loader: 'file-loader',
            options: {
              name: 'images/[name].[ext]',
            },
          },
        ]
      },
```

Reply:
@priscilawebdev my bad.

The webpack config is not in sync with the backend for web. 

Please update the `file-loader` config without options or without define the location of them, ALL files should be in the root of `static` files, otherwise return 404.
```
{
        test: /\.(jpe?g|png|gif|svg)$/,
        use: [
          {
            loader: 'file-loader'
          },
        ]
      },
```
That fix the issue. Let's webpack handle the names for us.

Reply:
@juanpicado ok 👍